vald/start.go

```diff
@@ -211,11 +211,6 @@ func listen(ctx context.Context, clientCtx sdkClient.Context, txf tx.Factory, ax
 		return cl, nil
 	})
 	tssMgr := createTSSMgr(bc, clientCtx, axelarCfg, logger, valAddr.String(), cdc)
-	if len(recoveryJSON) > 0 {
```

Comment:
Shouldn't we also get rid of `recoveryJSON` entirely from being passed into the function `listen`?